Source/player.cpp

```diff
@@ -1943,6 +1943,41 @@ void Player::RestorePartialLife()
 	_pHPBase = std::min(_pHPBase + l, _pMaxHPBase);
 }
 
+void Player::RegenLife()

```

Comment:
FYI, it looks like your life/mana regen code slipped into this PR.

Reply:
It sure did